webpages/js/formulas.js

```diff
@@ -238,13 +238,13 @@
         id: 'forestPlotPercentAggr',
         label: 'Forest Plot (percentages)',
         func: forestPlotPercentAggr,
-        parameters: [ 'group 1 (e.g. experimental percentage)', 'group 1 N', 'group 2 (e.g. control percentage)', 'group 2 N' ],
+        parameters: [ 'group 1 (e.g. experimental percentage)', 'group 1 N', 'group 2 (e.g. control percentage)', 'group 2 N', 'moderator' ],
```

Comment:
please have "Forest Plot" without groups, and "Forest Plot With Groups", basically the work done here becomes Forest plot with groups, and we need to copy a historic version of Forest plot for the without groups.

In effect, we'd have 4 (not just 2) forestPlot*Aggr functions here in formulas.js, and two different drawForestPlot functions in metaanalysis.js (we can leave refactoring for later)